lib/json_schema_form/umm_json_form.rb

```diff
@@ -120,6 +121,16 @@ def sanitize_form_input(input, form_id, current_value = {})
     input
   end
 
+  # TODO: Remove when UMM_S 1.3.2 is implemented.
```

Comment:
This is a hack to get us around the schema capitalization difference between UMM-T and UMM-S. I hope to remove this in sprint 4.